src/components/Input/FormInput/FormInputs.style.js

```diff
@@ -1,10 +1,9 @@
 import styled from "styled-components";
-import { ReactComponent as XIcon } from "assets/icon/ic_X.svg";
 
 export const FormInput = styled.input`
   width: 100%;
   height: 56px;
-  background-color: var(--input-bg);
+  background-color: var(--cool-gray300);

```

Comment:
styled components를 쓰고계시니, 이런 value들도 루트에 css vars로 정의해둔다기보다는 상수를 담은 객체에서 편하게 꺼내쓰는 방식은 어떨까요? 

이런식으로 값을 범주화하는 namespace를 만들어두어 프로그램 내에서 여기저기 활용하실수있답니다 :) 

```ts
export const colors = {
 coolGray300: ''
}
```

```suggestion
  background-color: ${colors.coolGray300};
```

몇개 예시를 더 들어볼게요.

```ts
// zIndex를 범주화
export const zIndexStyle = {
  base: 1,
  nav: 2,
  floating: 1000,
};

// z-index: ${zIndexStyle.base + 1}
```
```ts
// mediaQueries를 범주화
const BREAKPOINT_TABLET = 768;
const BREAKPOINT_PC = 1200;

export const onTablet = `@media only screen and (min-width: ${BREAKPOINT_TABLET}px) and (max-width: ${
  BREAKPOINT_PC - 1
}px)`;

export const onPc = `@media only screen and (min-width: ${BREAKPOINT_PC}px)`;

export const onTabletAndPc = `@media only screen and (min-width: ${BREAKPOINT_TABLET}px)`;

/*
${onTabletAndPC} {
    background-color: ${colors.coolGray300};
 }
 */
```